DSharpPlus.Commands/CommandsExtension.cs

```diff
@@ -20,6 +23,14 @@ namespace DSharpPlus.Commands;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
 
+using CheckFunc = System.Func
+<
+    object, 
+    DSharpPlus.Commands.ContextChecks.ContextCheckAttribute, 
+    DSharpPlus.Commands.Trees.CommandContext, 
+    System.Threading.Tasks.ValueTask<string?>
+>;
```

Comment:
Why are we using full namespaces here?

Reply:
While the `DSharpPlus.Commands` prefix isn't required, the rest still are for successful compilation. Keeping it as is.